lib/resources/r4/coverage.yaml

```diff
@@ -28,8 +28,8 @@ fields:
   note: Currently supports only `active`.
 
 - name: subscriber
-  required: 'No'
-  cardinality: 0..1
+  required: 'Yes'
+  cardinality: 1..1
```

Comment:
the cardinality should still reflect the HL7 cardinality. required indicates that the field is Cerner-required

Reply:
Reverted in https://github.com/cerner/fhir.cerner.com/pull/314/commits/5f38d48b35ae270096f12ecb6da889ff84d9086c